db/fns.md

```diff
@@ -470,6 +476,22 @@ Get github check run id and check suite id
 
 Get a single check from a task_id.
 
+### get_github_checks_by_task_group_id
+
+* *Mode*: read
+* *Arguments*:
+  * `page_size_in integer`
+  * `page_offset_in integer`
+  * `task_group_id_in text`
+* *Returns*: `table`
+  * `   task_group_id text`
```

Comment:
<img width="215" alt="Screenshot 2023-05-11 at 7 43 44 AM" src="https://github.com/taskcluster/taskcluster/assets/92693437/cb60606f-8145-48ae-8915-94437e1d6466">

Not sure it's truly an issue with whitespace or if it's how github's presenting the file, but this line looks more indented for some reason. Maybe there's some extra whitespace in db/versions/0085.yml?